app/frontend/src/components/bcgov/BCGovHeader.vue

```diff
@@ -1,29 +1,21 @@
 <template>
   <header :class="{'gov-header': true, 'd-print-none': formSubmitMode}">
-    <v-toolbar color="#003366" flat>
+    <v-toolbar color="#003366" flat class="px-md-12">
       <!-- Navbar content -->
       <a href="https://www2.gov.bc.ca" data-test="btn-header-logo">
         <v-img
           alt="B.C. Government Logo"
-          class="d-none d-sm-flex d-md-none"
-          contain
-          height="3.5rem"
-          src="@/assets/images/bc_logo_square.svg"
-          width="3.5rem"
-        />
-        <v-img
-          alt="B.C. Government Logo"
-          class="d-none d-md-flex"
+          class="d-flex"
           contain
           height="3.5rem"
           src="@/assets/images/bc_logo.svg"
           width="10rem"
         />
       </a>
-      <h1 v-if="!formSubmitMode" data-test="btn-header-title" class="font-weight-bold title">
+      <h1 v-if="!formSubmitMode" data-test="btn-header-title" class="font-weight-bold title d-none d-md-flex pl-md-4">
```

Comment:
This can probably just be pl-4 - don't need to specify specific viewport size for padding when it only shows or hides.